tests/seta/test_models.py

```diff
@@ -0,0 +1,89 @@
+import datetime
+
+from django.db.utils import IntegrityError
+from django.utils import timezone
+import pytest
+
+from treeherder.seta.models import (JobPriority,
+                                    TaskRequest)
+
+TOMORROW = timezone.now() + datetime.timedelta(days=1)
+YESTERDAY = timezone.now() - datetime.timedelta(days=1)
+slow = pytest.mark.slow
+
+
+# TaskRequest tests
+@slow
+def test_create_instance(test_repository):
+    TaskRequest.objects.create(repository=test_repository,
+                               counter=0,
+                               last_request=timezone.now(),
+                               reset_delta=0)
+
+
+# JobPriority tests
+def test_expired_job_priority():
+    jb = JobPriority(testtype='web-platform-tests-1',
+                     buildtype='opt',
+                     platform='windows8-64',
+                     priority=1,
+                     timeout=5400,
+                     expiration_date=YESTERDAY,
+                     buildsystem='taskcluster')
+    assert jb.has_expired() is True
```

Comment:
I meant "jp" as job priority. I don't want to use job when speaking of job priorities as I want it to be clear that it is referring to an instance of the model.

We use job around the code base and I want to differentiate it by using job_priority or jp.